src/Easydropdown/Easydropdown.test.ts

```diff
@@ -152,4 +152,30 @@ describe('Easydropdown', () => {
             assert.equal(edd.dom.item.length, 7);
         });
     });
+
+    describe('.validate()', () => { 
```

Comment:
Please create two additional test cases for a required and non-required select with no "placeholder" option.